src/routes/PetsPage/PetsPage.js

```diff
@@ -1,34 +1,200 @@
 import React, { Component } from 'react';
+import Select from 'react-select';
 import PetCard from '../../components/PetCard/PetCard';
-import { Section } from '../../components/Utils/Utils';
+import { Input, Section } from '../../components/Utils/Utils';
 import PetsService from '../../services/petsService';
 
 export default class PetsPage extends Component {
```

Comment:
Is there any way to call this `PetsPage` from `http://localhost:3000/shelters/shelterID` page? The shelter page also has pet list and if would be nice if we can just call `PetsPage`  from the shelter page with the shelter pets data. I really like the way you set the filters and it would be nice if we can use it in the below page!
![image](https://user-images.githubusercontent.com/71732016/153928098-f03bc872-33a6-4898-9da3-829a340c659e.png)

Reply:
Yes I should be able to change it to a component!

Reply:
Thank you! I am excited to see your change!!